src/Widget/Field.php

```diff
@@ -157,6 +157,10 @@ public function label(bool $enabledLabel = true, array $options = [], ?string $l
             $new->labelOptions['label'] = $label;
         }
 
+        if ($new->data->isAttributeRequired($new->attribute)) {
+            Html::addCssClass($options, $new->requiredCssClass);
```

Comment:
Maybe it's a better idea to add HTML5 validation attribute here? What do you think?

See https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required

Reply:
That would be an attribute on the input, so browser does validation. I do class on label for visual indication for end user of required field. Different use case.

Reply:
` aria-required="true"` in the input is already btw